.analysis_options

```diff
@@ -6,13 +6,23 @@ analyzer:
 linter:
   rules:
 #    - always_declare_return_types
+#    - always_put_control_body_on_new_line
+    - always_require_non_null_named_parameters
```

Comment:
I enable all the new rules except that because it had a recurrent false positive that I reported and @a14n fixed quickly, I will enable it now